### PR TITLE
refactor(api): put rate model rpc call into multicall

### PR DESCRIPTION
**Pull request by dohaki**
Closes ACX-2053

### CODE REVIEW

File: api/suggested-fees.ts [hunk 1]
```diff
@@ -214,21 +214,26 @@ const handler = async (
...
-          },
-          computedOriginChainId,
-          destinationChainId
-        ),
         getCachedTokenPrice(l1Token, baseCurrency),
       ]);
+    const parsedL1TokenConfig =
+      sdk.contracts.acrossConfigStore.Client.parseL1TokenConfig(
+        String(rawL1TokenConfig)
+      );
```
Comment (james-a-morris) [inline api/suggested-fees.ts:232]: Is the `rawL1TokenConfig` an encoded value?

**Review by james-a-morris — COMMENTED** [review]
One comment

File: api/suggested-fees.ts (hunk 1 above)
Comment (dohaki) [reply api/suggested-fees.ts:232]: It is a stringified JSON

**Review by james-a-morris — APPROVED** [review]

**Review by pxrl — APPROVED** [review]
LGTM - very straightforward.

**Pull request merged**